src/store/ui-navigation-module/state.ts

```diff
@@ -1,13 +1,15 @@
 export interface UINavModule {
   isShoww: boolean;
   isShow: boolean;
+  addDoc: boolean;
```

Comment:
uiNav may not be a valid Vuex module because its ui related rather than business related